### DISCUSSION TITLE
return the user's id the first time they chat with the bot.

### DISCUSSION THREAD

**Discussion by noriellecruz** [n3d1117/chatgpt-telegram-bot#363] [Ideas]
@n3d1117 i think this is a must-have feature. instead of getting their id from getidsbot, they can just copy the id returned in the first response of the bot. it will save time for both admin and the user.

i already implemented this on my branch. i can create a pull request if anyone thinks this is a good idea.

**Comment by n3d1117** [comment]
I like this, I've been using logs to get ids but I agree it's not the best. Please go ahead with the PR!
